docs/dotnet/dotnet-aspnetcore-metrics.md

```diff
@@ -21,6 +21,19 @@ This article defines semantic conventions for ASP.NET Core metrics.
   - [Metric: `aspnetcore.rate_limiting.queued_requests`](#metric-aspnetcorerate_limitingqueued_requests)
   - [Metric: `aspnetcore.rate_limiting.request.time_in_queue`](#metric-aspnetcorerate_limitingrequesttime_in_queue)
   - [Metric: `aspnetcore.rate_limiting.requests`](#metric-aspnetcorerate_limitingrequests)
+- [Identity](#identity)
+  - [Metric: `aspnetcore.identity.user.create`](#metric-aspnetcoreidentityusercreate)
```

Comment:
I should have asked before, but is there a reason we report them as counters and not duration histograms?

Reply:
We can still create histograms with 1-2 buckets by default, the benefit is that it's comparable to counters in terms of volume, but users can change bucket boundaries when they care about durations. Not blocking here, FJYI for the future use-cases

Reply:
I've changed create/update/delete to histograms.